modules/authorization/policy-set-definition/README.md

```diff
@@ -571,3 +487,90 @@ module policySetDefinition './authorization/policy-set-definition/main.bicep' =
 
 </details>
 <p>
+
+
+## Notes
+
+## Module Pre-requisites
```

Comment:
```suggestion
### Module Pre-requisites
```

This I have to say the statement itself has not really anything to do with the module but is a general service guidance

Reply:
I've removed the whole section.